client/Makefile

```diff
@@ -76,19 +72,14 @@ test:
 PHONY: test-all
 test-all:
 	$(MAKE) test-audit
-	$(MAKE) test-format
```

Comment:
don't remove` test-format` here. If the format check includes cypress, is a little weird, but less annoying then losing the `format` commands inside `client` all together